fighthealthinsurance/models.py

```
@@ -2216,6 +2216,95 @@ def __str__(self):
 
+class PolicyDocument(ExportModelOperationsMixin("PolicyDocument"), models.Model):  # type: ignore
+    """
+    Stores uploaded policy documents (Summary of Benefits, Medical Policy PDFs).
+    Used to help users understand their insurance coverage.
+    """
+
+    DOCUMENT_TYPE_CHOICES = [
+        ("summary_of_benefits", "Summary of Benefits"),
+        ("medical_policy", "Medical Policy"),
+        ("other", "Other Policy Document"),
+    ]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)